src/commands/radio/search.js

```diff
@@ -0,0 +1,47 @@
+const { SearchCommand, CommandError, SwitchbladeEmbed } = require('../../')
+
+module.exports = class TuneIn extends SearchCommand {
+  constructor (client) {
+    super(client, {
+      name: 'tunein',
+      category: 'radio',
+      parameters: [
+        {
+          type: 'string',
+          full: true
+        }
+      ],
+      requirements: { apis: ['tunein'] }
+    })
+  }
+
```

Comment:
Missing the `searchResultFormatter` method. please look [here](https://github.com/SwitchbladeBot/switchblade/blob/dev/src/commands/misc/spotify/album.js#L18-L20) for an example